python/tvm/auto_scheduler/compute_dag.py

```diff
@@ -50,6 +50,11 @@ class ComputeDAG(Object):
     compute : Union[List[Tensor], str, Schedule]
         Input/output tensors or workload key for a compute declaration.
     """
+    LAYOUT_REWRITE_TABLE = {
+        "NoRewrite": 0,
+        "RewriteWithPlaceholder": 1,
+        "RewriteWithPreTranspose": 2,
+    }
```

Comment:
Is that possible to avoid duplicated layout rewrite option tables, although to be honest I have no idea how to do that...

Reply:
Why do we use strings? Can we just use variables like this?
https://github.com/apache/incubator-tvm/blob/8d56164107ac2be9f2f53f13de202a7bbf1c1c66/python/tvm/auto_scheduler/measure.py#L464-L472